Netimobiledevice/HelperFiles/ModelIdentifier.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Netimobiledevice.HelperFiles
+{
+    public static class ModelIdentifier
+    {
+        public static string GetDeviceModelName(string identifier)
+        {
+            Dictionary<string, string> modelMapping = new Dictionary<string, string>
```

Comment:
I know they are discontinued now but iPods in theory would be able to work with this so probably should add the models for them in as well.

Reply:
Yes, Currently, I have this list and if anything found, will add later to this list.